Source/DafnyCore/AST/Grammar/ProgramParser.cs

```diff
@@ -102,6 +104,23 @@ public class ProgramParser {
 
     ShowWarningsForIncludeCycles(program);
 
+    compilation.TranslationRecord = new TranslationRecord();
+    var records = options.Get(CommonOptionBag.TranslationRecords);
+    if (records != null) {
+      foreach (var path in options.Get(CommonOptionBag.TranslationRecords)) {
+        using var reader = new StreamReader(path.FullName);
```

Comment:
Can use `async using`

Reply:
Actually I can't: for some reason `TextReader` does not implement `IAsyncDisposable`, even though `TextWriter` does.